qiskit/optimization/algorithms/minimum_eigen_optimizer.py

```diff
@@ -148,7 +163,9 @@ def solve(self, problem: QuadraticProgram) -> MinimumEigenOptimizationResult:
         self._verify_compatibility(problem)
 
         # convert problem to QUBO
-        problem_ = self._qubo_converter.convert(problem)
+        problem_ = problem
+        for converter in self._converters:
+            problem_ = converter.convert(problem_)
```

Comment:
Similar for this code though I agree its less that would get a converted problem given the set of converters. Just seeing the same thing in many places begs the question would be be better to have common methods and the above (and other similiar)  become a single call to the common method

Reply:
I added `_converter()` to `OptimizationAlgorithm` and now, the code seems pretty nice like `problem_ = self._convert(problem, self._converters)`. However, I'm stuck in a circular import. If I import converters in `OptimizationAlgorithm`, I get a circular import error... Is there any better way to import?